src/Makefile.am

```diff
@@ -572,7 +576,19 @@ bitcoind_LDADD = \
   $(LIBMEMENV) \
   $(LIBSECP256K1)
 
-bitcoind_LDADD += $(BOOST_LIBS) $(BDB_LIBS) $(MINIUPNPC_LIBS) $(EVENT_PTHREADS_LIBS) $(EVENT_LIBS) $(ZMQ_LIBS)
+bitcoind_common_ldadd += $(BOOST_LIBS) $(BDB_LIBS) $(MINIUPNPC_LIBS) $(EVENT_PTHREADS_LIBS) $(EVENT_LIBS) $(ZMQ_LIBS) $(LIBMULTIPROCESS_LIBS)
```

Comment:
Why is LIBMULTIPROCESS_LIBS neeeded for the single-binary `bitcoind`?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/16367#discussion_r396726227

> Why is LIBMULTIPROCESS_LIBS neeeded for the single-binary `bitcoind`?

It's not needed, removed entirely from this PR and removed from bitcoind link line in #10102